viscy/data/hcs.py

```diff
@@ -602,7 +602,8 @@ def _train_transform(self) -> list[Callable]:
         else:
             self.augmentations = []
         if z_scale_range is not None:
-            if isinstance(z_scale_range, float):
+            if isinstance(z_scale_range, (float, int)):
+                z_scale_range = float(z_scale_range)
```

Comment:
Should this change be merged with this PR?

Reply:
I dont see why not.